enhancements/management-workload-partitioning.md

```diff
@@ -0,0 +1,848 @@
+workloads. Future work will eliminate the need for the extra operator
+(see GA graduation criteria).
+
+### User Stories
+
+#### Radio Access Network (RAN) Use Case
+
+In the context of telcocommunications service providers' 5G Radio
+Access Networks, it is increasingly common to see "cloud native"
+implementations of the 5G Distributed Unit (DU) component. Due to
+latency constraints, this DU component needs to be deployed very close
+to the radio antenna for which it is responsible. In practice, this
+can mean running this component on anything from a single server at
+the base of a remote cell tower or in a datacenter-like environment
+serving several base stations.
+
+A hypothetical DU example is an unusually resource-intensive workload,
+requiring 20 dedicated cores, 24 GiB of RAM consumed as huge pages,
+multiple SR-IOV NICs carrying several Gbps of traffic each, and
+specialized accelerator devices. The node hosting this workload must
+run a realtime kernel, be carefully tuned to ensure low-latency
+requirements can be met, and be configured to support features like
+Precision Timing Protocol (PTP).
+
+The most constrained resource for RAN deployments is CPU, so for now
+the focus of this enhancement is managing CPU requirements.
+Kubernetes resource requests for CPU time are measured in fractional
+"[Kubernetes
+CPUs](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu)". For
+bare metal, 1 Kubernetes CPU is equivalent to 1 hyperthread.
+
+Due to the resource-intensive workload, the overhead for platform
+components such as the OpenShift platform and add-on operators is
+severely restricted when compared to other uses. For example, one
+customer has allocated 4 hyperthreads for all components other than
+their own workloads. OpenShift currently requires 7.
+
+### Implementation Details/Notes/Constraints
+
+#### High-level End-to-end Workflow
+
+This section outlines an end-to-end workflow for deploying a cluster
+with workload partitioning enabled and how pods are correctly
+scheduled to run on the management CPU pool.
+
+1. User sits down at their computer.
+2. The user creates a machine config manifest to configure CRI-O to
+   partition management workloads.
+3. The user creates a machine config manifest to write a configuration
+   file for kubelet to enable the same workload partition. The file
+   should only be readable by the kubelet.
+4. The user runs the installer to create the standard manifests, adds
+   their extra manifests from steps 2 and 3, then creates the cluster.
+5. The kubelet starts up and finds the configuration file enabling the
+   new feature.
+6. The kubelet advertises `management.workload.openshift.io/cores`
+   extended resources on the node based on the number of CPUs in the
+   host.
+7. The kubelet reads static pod definitions. It replaces the `cpu`
+   requests with `cpu.workload.openshift.io/management` requests of
```

Comment:
`cpu.workload.openshift.io/management` -> `management.workload.openshift.io/cores`

Reply:
Thanks, #728 fixes this